source/jsonnet/england-wales/individual/blocks/identity-and-health/ethnic_group_asian_other.jsonnet

```diff
@@ -20,25 +29,18 @@ local question(title, region_code) = (
   }
 );
 
-local nonProxyTitle = 'You selected “Any other Asian background”. How would you describe your Asian or Asian British ethnic group or background?';
-local proxyTitle = {
-  text: 'You selected “Any other Asian background”. How would <em>{person_name}</em> describe their Asian or Asian British ethnic group or background?',
-  placeholders: [
-    placeholders.personName(),
-  ],
-};
 
-function(region_code) {
+{
   type: 'Question',
   id: 'other-asian-or-asian-british-ethnic-group',
-  page_title: 'Other Asian or Asian British ethnic group or background',
+  page_title: 'Other Asian ethnic group or background',
```

Comment:
I don't see this change specced on the relevant tab of the google doc?

Reply:
It was something that was missed. Ill add it to the docs.